src/js/wrapper.ts

```diff
@@ -279,21 +280,23 @@ export const NATIVE: SentryNativeWrapper = {
       throw this._NativeClientError;
     }
 
-    // separate and serialze all non-default user keys.
-    let defaultUserKeys = null;
-    let otherUserKeys = null;
+    // separate and serialize all non-default user keys.
+    let userKeys = null;
+    let userDataKeys = null;
     if (user) {
-      const { id, ip_address, email, username, ...otherKeys } = user;
-      defaultUserKeys = this._serializeObject({
-        email,
+      const { id, ip_address, email, username, segment, ...otherKeys } = user;
```

Comment:
```suggestion
      const { id, ip_address, email, username, segment, ...dataKeys } = user;
```

So we get rid of `other`?

Reply:
I just renamed all the variables that at the end are user data, just so it's clear what the variables represents.